### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by redevilable**
repaired a bug from (i==3) to (n==9)

### CODE REVIEW

**Review by EricHo233 — COMMENTED** [review]
the change is ok

File: mcr.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@ int  main(){
 				break; // need to terminate the problem
 			}
 		}
-	if (i==3) // all celles with i=0 to 2 have been inputted above but no winner yet
+	if (n==9) // all celles with i=0 to 2 have been inputted above but no winner yet
```
Comment (Salvatores71) [inline mcr.cpp:45]: 超出n的作用域，错误的

Comment (Salvatores71) [reply mcr.cpp:45]: ！isWin（game）

Comment (redevilable) [reply mcr.cpp:45]: i received it, let me modify

Comment (EricHo233) [reply mcr.cpp:45]: change i==3 to isWin(game)

**Review by StephenWang3 — CHANGES_REQUESTED** [review]
应该把n的定义放在for循环外面

**Pull request merged**